### DISCUSSION TITLE
How is the trends place sorted?

### DISCUSSION THREAD

**Discussion by XzavierDunn** [tweepy/tweepy#1213] [Questions] [answered]
I was getting global trending topics with the trends_place function and I noticed they weren't sorted by number of tweets so I was wondering if that was a goof somewhere on tweepy's end or what was going on there. So less of an issue more of a general question but I was wondering if anyone knew the answer.

**Comment by Harmon758** [comment, marked as answer]
They're returned in the order that the Twitter API provides for that endpoint.
The `tweet_volume` returned is for the last 24 hour, which is not necessarily how it's sorted.
See https://developer.twitter.com/en/docs/trends/trends-for-location/api-reference/get-trends-place and https://twittercommunity.com/t/is-there-any-easy-way-to-get-top-trending-topics/104746/6 for more information.
